zerver/apps.py

```diff
@@ -8,17 +8,15 @@
 import logging
 
 
-def flush_cache(sender, **kwargs):
-    # type: (AppConfig, **Any) -> None
+def flush_cache(sender: AppConfig, **kwargs: **Any) -> None:
```

Comment:
I believe this will just be `**kwargs: Any`.

Reply:
I see, right, I was wondering of the superfluous `**` as well.

Reply:
Fixed with `.replace('**', '')`

Reply:
Also added `.replace('*', '')` after that replace.